packages/client/src/RepositoryClient.ts

```diff
@@ -80,7 +85,7 @@ export class RepositoryClient {
         return this
     }
 
-    withClientIdAndrepository(id: string, repository: string): RepositoryClient {
+    withClientIdAndRepository(id: string, repository: string): RepositoryClient {
```

Comment:
Shouldn't `repository` be explicitly allowed to be `null` here?

Reply:
Yes, you are right. Fixed